docs/app/Components/ComponentDoc/ComponentExample/ComponentExample.js

```diff
@@ -442,18 +437,10 @@ class ComponentExample extends Component {
             </Grid.Column>
           </Grid.Row>
 
-          <Grid.Row columns={1}>
-            {children && (
-              <Grid.Column style={childrenStyle}>
-                {children}
-              </Grid.Column>
-            )}
-          </Grid.Row>
+          <Grid.Row columns={1}>{children && <Grid.Column style={childrenStyle}>{children}</Grid.Column>}</Grid.Row>
```

Comment:
Perhaps we should shorten our line length.  This is quite hard to read.

```jsx
<Grid.Row columns={1}>{children && <Grid.Column style={childrenStyle}>{children}</Grid.Column>}</Grid.Row>
```

This line is 118 columns, maybe go with 100 columns?

```jsx
<Grid.Row columns={1}>
  {children && <Grid.Column style={childrenStyle}>{children}</Grid.Column>}
</Grid.Row>
```